cmd/orchestrator.go

```diff
@@ -233,7 +233,9 @@ func orchestratorStart(r *reporter.Reporter, config OrchestratorConfiguration, d
 		httpComponent,
 		clickhouseDBComponent,
 		clickhouseComponent,
-		kafkaComponent,
+	}
+	if kafkaComponent != nil {
+		components = append(components, kafkaComponent)
```

Comment:
I don't think this is needed. A component can be nil, as long as Start()/Stop() can handle it.